README.md

```diff
@@ -18,6 +25,18 @@ sudo gem install govuk-connect
 
 ## Usage
 
-```bash
-govuk-connect --help
+If you have the gds-cli installed (you [should do](https://docs.publishing.service.gov.uk/manual/access-aws-console.html)!), you can use this tool within it. This means you only have to use one tool to do AWS and SSH.
+
 ```
+gds govuk connect --help
+```
+
+It can be shortened to `gds govuk c`.
+
+## Developing the gem
+
+TODO: Someone who has made changes to this should write some words here.
```

Comment:
Do we need this? Perhaps we should add it when we've experienced issues with development that merit some docs.

Reply:
Sure.